code/ObjFileImporter.cpp

```diff
@@ -138,8 +138,8 @@ void ObjFileImporter::InternReadFile( const std::string& pFile, aiScene* pScene,
     if ( pos != std::string::npos ) {
         modelName = pFile.substr(pos+1, pFile.size() - pos - 1);
         folderName = pFile.substr( 0, pos );
-        if ( folderName.empty() ) {
-            pIOHandler->PushDirectory( folderName );
+        if ( !folderName.empty() ) {
+            pIOHandler->PushDirectory( folderName + "/");
```

Comment:
Looked through the code and maybe this `+ "/"` should be placed in the ObjFileParser.cpp as there is the place where path is added:
```
...
    if ( m_pIO->StackSize() > 0 ) {
        const std::string &path = m_pIO->CurrentDirectory();
        absName = path + strMatName;
    } else {
        absName = strMatName;
    }
    IOStream *pFile = m_pIO->Open( absName );
...
```

What do You think ?